src/libraries/Microsoft.PowerFx.Interpreter/Functions/Library.cs

```diff
@@ -1628,14 +1628,14 @@ static Library()
                 BuiltinFunctionsCore.Trunc,
                 StandardErrorHandling<FormulaValue>(
                     BuiltinFunctionsCore.Trunc.Name,
-                    expandArguments: InsertDefaultValues(outputArgsCount: 2, fillWith: new NumberValue(IRContext.NotInSource(FormulaType.Number), 0)),
+                    expandArguments: NoArgExpansion,

```

Comment:
>NoArgExpansion [](http://example.com/codeflow?start=37&length=14)

`Trunc` function will have its own target function and wont need default values being injected.